src/main/java/com/ongvida/domain/repository/ProfessorRepository.java

```diff
@@ -1,15 +1,14 @@
 package com.ongvida.domain.repository;
 
 import java.util.List;
-
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 import com.ongvida.domain.model.Professor;
 
 @Repository
-
 public interface ProfessorRepository extends JpaRepository<Professor,Long>{
 
 	List<Professor> findByNome(String nome);
```

Comment:
Detalhe aqui é que como tu vai mudar o "nome" para "name" na Entity Professor -> você precisa alterar a consulta.  Quem faz essa consulta pra ti é o Spring Data JPA, ele pega à partir do nome que tu passa. Logo, se o atributo for name , o método vai precisar ser findByName

```suggestion
	List<Professor> findByName(String name);
```